containers/src/Data/IntMap/Internal.hs

```diff
@@ -294,7 +294,7 @@ module Data.IntMap.Internal (
     ) where
 
 import Data.Functor.Identity (Identity (..))
-import Control.Applicative (liftA2)
+import Control.Applicative hiding (empty)
```

Comment:
Using `hiding` isn't best practice.  Please change to explicit import list.
(The exception is `import Prelude hiding (...)` because explicit import lists for `Prelude` are looong.)

On a second thought, I think that placing the `import Control.Applicative (liftA2)` under a `!MIN_VERSION_base` constraint, and not changing anything else, would be the best remedy here.